IHP/Log.hs

```diff
@@ -0,0 +1,77 @@
+{-|
+Module: IHP.Log.Logging
+Description:  Functions to write logs at all log levels.
+
+Import this module qualified!
+
+-}
+module IHP.Log (
+  module IHP.Log.Types,
+  debug,
+  info,
+  warn,
+  error,
+  fatal,
+  unknown,
+  makeRequestLogger,
+  defaultRequestLogger
+) where
+
+import IHP.HaskellSupport hiding (debug)
+
+import qualified Prelude
+import CorePrelude hiding (putStr, putStrLn, print, error, show, log, debug)
+import Control.Monad (when)
+import Data.Text as Text
+import IHP.Log.Types
+import Network.Wai (Middleware)
+import Network.Wai.Middleware.RequestLogger (mkRequestLogger, RequestLoggerSettings, destination)
+import qualified Network.Wai.Middleware.RequestLogger as RequestLogger
+import Data.Default.Class (Default (def))
+import Data.String.Conversions (cs)
+import System.IO.Unsafe (unsafePerformIO)
+
+-- | Format a log and send it to the logger.
```

Comment:
Can we add a code example here? :) These show up e.g. in IDE's, so it's good when most of the primaryly used functions have code examples